### PR TITLE
split log bundle

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: LogBundle/Controller/LogController.php [hunk 1]
```diff
@@ -32,7 +32,7 @@ public function listAction(Request $request)
     {
         $repository =  $this->get('open_orchestra_log.repository.log');
         $configuration = PaginateFinderConfiguration::generateFromRequest($request);
-        $mapping = $this->get('open_orchestra_base.annotation_search_reader')->extractMapping('OpenOrchestra\LogBundle\Document\Log');
+        $mapping = $this->get('open_orchestra_base.annotation_search_reader')->extractMapping($this->container->getParameter('open_orchestra_log.document.log.class'));
```
Comment (itkg-alavieille) [inline LogBundle/Controller/LogController.php]: i tkink is ``open_orchestra_model_log`` and not ``open_orchestra_log``

File: ModelLogBundle/DependencyInjection/OpenOrchestraModelLogExtension.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+     * @param ContainerBuilder $container
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $configuration = new Configuration();
+        $config = $this->processConfiguration($configuration, $configs);
+
+        foreach ($config['document'] as $class => $content) {
+            if (is_array($content)) {
+                $container->setParameter('open_orchestra_log.document.' . $class . '.class', $content['class']);
```
Comment (itkg-alavieille) [inline ModelLogBundle/DependencyInjection/OpenOrchestraModelLogExtension.php]: same it's ``open_orchestra_model_log``

File: ModelLogBundle/DependencyInjection/OpenOrchestraModelLogExtension.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                $definition->setFactory(array(new Reference('object_manager'), 'getRepository'));
+                $definition->addMethodCall('setAggregationQueryBuilder', array(
+                    new Reference('doctrine_mongodb.odm.default_aggregation_query')
+                ));
+                if (method_exists($content['repository'],'setFilterTypeManager')) {
+                    $definition->addMethodCall('setFilterTypeManager', array(
+                        new Reference('open_orchestra_pagination.filter_type.manager')
+                    ));
+                }
+                $container->setDefinition('open_orchestra_log.repository.' . $class, $definition);
```
Comment (itkg-alavieille) [inline ModelLogBundle/DependencyInjection/OpenOrchestraModelLogExtension.php]: same

**Comment by nicolasThal** [comment]
I left the prefix open_orchestra_log and not open_orchestra_model_log because the second prefix will give the impression that you have to create a ModelLogBundle.

In fact, you could just create a FooBundle and declare the services linked to the LogBundle

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**